infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/DockerInternalRuntime.java

```diff
@@ -203,29 +208,39 @@ protected void internalStart(Map<String, String> startOptions) throws Infrastruc
             }
         } catch (InfrastructureException | InterruptedException | RuntimeException e) {
```

Comment:
no, I think it can be like it was previously because at some point it simplifies the handling of interrupt and explicitly indicates that the call is blocking.